starters/basic-starter/components/node--event.tsx

```diff
@@ -13,8 +13,7 @@ export function NodeEvent({ node, ...props }) {
             <a className="block overflow-hidden no-underline rounded-md">
               <MediaImage
                 media={node.field_event_image}
-                priority
-                sizes="(min-width: 968px) 410px, (min-width: 768px) 50vw, 100vw"
```

Comment:
Wouldn't it make sense to keep the priority and sizes attribute here even though we are requesting an image style?

Reply:
Oh ok I wasn't sure if image styles would still use those properties. Added back